source/common/jwt_authn/extractor.cc

```diff
@@ -0,0 +1,148 @@
+#include "common/jwt_authn/extractor.h"
+
+#include "common/common/utility.h"
+#include "common/http/utility.h"
+
+using ::envoy::config::filter::http::jwt_authn::v2alpha::JwtAuthentication;
+
+namespace Envoy {
+namespace JwtAuthn {
+namespace {
+
+// The authorization header name.
+const std::string AuthorizationHeader = "authorization";
+
+// The authorization bearer prefix.
```

Comment:
Putting those constants inside a singleton under same convention as
https://github.com/envoyproxy/envoy/blob/master/source/common/http/headers.h#L27

This will avoid static initialization order fiasco

Reply:
Done